gradle.properties

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-ideaEdition = IU
+ideaEdition = IC
```

Comment:
lets leave this as IU, no?

Reply:
Sure, just was making sure we're building with IC in Kokoro too, I'll revert it before merge.

Reply:
sgtm